psydac/api/fem.py

```diff
@@ -372,42 +440,89 @@ def get_space_indices_from_target(self, domain, target):
                 j = i
         return i,j
 
-    def construct_arguments(self, backend=None):
+    def construct_arguments(self, backend=None, with_openmp=False):
 
         test_basis, test_degrees, spans, pads = construct_test_space_arguments(self.test_basis)
         trial_basis, trial_degrees, pads      = construct_trial_space_arguments(self.trial_basis)
         n_elements, quads, quad_degrees       = construct_quad_grids_arguments(self.grid[0], use_weights=False)
         if len(self.grid)>1:
             quads  = [*quads, *self.grid[1].points]
 
-        pads                      = self.test_basis.space.vector_space.pads
-        global_mats               = self.allocate_matrices(backend)
-        self._global_matrices     = [M._data for M in global_mats]
+        pads                  = self.test_basis.space.vector_space.pads
+        global_mats           = self.allocate_matrices(backend)
```

Comment:
Please move this call to the `__init__`

Reply:
done in [ec10ae7](https://github.com/pyccel/psydac/pull/213/commits/ec10ae70e2106c6cc339a5aa2ae5ac81c351a7f6)